src/component/slot/slot.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const StyledSlot = styled.div`
+  height: 175px;
+  position: relative;
+  display: inline-block;
+  margin-left: 165px;
+  transform: rotate(120deg);
+  transform-origin: 50% 100%;
+
+  &::after {
+    content: ${({ number }) => `"${number}"`};
```

Comment:
You probably don't need double quotes inside ticks. You can try to remove double quotes and see if works without them.

Reply:
I'll try now. Thank you, @tandaraelena!

Reply:
I tried. Without double quotes or single quotes, the number is not displayed on the triangle